security/advancedtls/advancedtls_integration_test.go

```diff
@@ -0,0 +1,509 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package advancedtls
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"log"
+	"net"
+	"sync"
+	"testing"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	pb "google.golang.org/grpc/examples/helloworld/helloworld"
+	"google.golang.org/grpc/security/advancedtls/testdata"
+)
+
+var (
+	address = "localhost:50051"
+	port    = ":50051"
+)
+
+// stageInfo contains a stage number indicating the current phase of each integration test, and a mutex.
```

Comment:
I know Go doesn't have limits on number of characters in lines but at least limiting the comments to 80 chars might make them easy to read. `credentials/alts` does this already but I will leave the decision to you and @dfawley.